game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -1142,35 +1124,38 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
         navalBombardment.execute(stack, bridge);
       }
     };
-    if (firstRun) {
-      steps.add(
-          new IExecutable() {
-            private static final long serialVersionUID = 3389635558184415797L;
+    // Removed in 2.1
+    new IExecutable() {
+      private static final long serialVersionUID = 3389635558184415797L;
 
-            @Override
-            public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
-              removeNonCombatants(bridge);
-            }
-          });
-      steps.add(
-          new LandParatroopers() {
-            private static final long serialVersionUID = 7193353768857658286L;
+      @Override
+      public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
+        final BattleStep removeNonCombatants = new RemoveNonCombatants(MustFightBattle.this);
+        removeNonCombatants.execute(stack, bridge);
+      }
+    };
+    // Removed in 2.1
+    new IExecutable() {
+      private static final long serialVersionUID = 7193353768857658286L;
 
-            @Override
-            public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
-              landParatroops(bridge);
-            }
-          });
-      steps.add(
-          new IExecutable() {
-            private static final long serialVersionUID = -6676316363537467594L;
+      @Override
+      public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
+        final BattleStep landParatroopers =
```

Comment:
style-nit, the intermediate variable is not really needed here.

EG:
```
 new LandParatroopers(MustFightBattle.this, MustFightBattle.this)
        .execute(stack, bridge);
```
Do you have a strong preference for the intermediate variable?

Reply:
No.  I just did it that way the first time and have kept all the other ones similar.  I can switch the style easily.